nautobot/users/api/urls.py

```diff
@@ -10,6 +10,8 @@
 
 # Saved Views
 router.register("saved-views", views.SavedViewViewSet)
+router.register("user-to-savedviews", views.UserToSavedViewViewSet)
```

Comment:
```suggestion
router.register("user-to-saved-views", views.UserToSavedViewViewSet)
```

Or should we follow other patterns and call this model and endpoint something like `UserSavedViewAssociation` instead?